Importer/Event/Error/ExceptionEvent.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Netdudes\ImporterBundle\Importer\Event\Error;
+
+use Netdudes\ImporterBundle\Importer\Log\LogInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+class ExceptionEvent extends Event
+{
+    /**
+     * @var LogInterface
+     */
+    protected $log;
+
+    /**
+     * @var \Exception
+     */
+    protected $exception;
+
+    /**
+     * @var bool
+     */
+    protected $stopped = false;
+
+    /**
+     * @var int|null
+     */
+    protected $index;
+
+    /**
+     * @param LogInterface $log
+     * @param \Exception   $exception
+     * @param int|null     $index
```

Comment:
Well the goal of last commit was to reduce the number of events. Your suggestion would make `ExceptionSubscriber` and `ExceptionEvent` more clear, but then we would have 3 events instead of 2.

What do you think @yakobe ?